source/src/iWalker.kt

```diff
@@ -1,3 +1,29 @@
 interface iWalker {
+    val curSpeed: Int
     fun walk()
-}
\ No newline at end of file
+    fun stop()
+    fun currentSpeed()
+    fun printAllInfo()
+}
+
+class ZombieWalker(): iWalker {
+    override val curSpeed: Int
+        get() = 1
```

Comment:
Getter should return the variable. If u wanna always return 1, u should make default value